dali/base/dasess.cpp

```diff
@@ -151,6 +151,26 @@ static CriticalSection sessionCrit;
 #define CLDAPE_getpermtimeout (-1)
 #define CLDAPE_ldapfailure    (-2)
 
+#ifdef NULL_DALIUSER_STACKTRACE
+
+static void logNullUser(const char * func, int line = __LINE__)
+{
+    const double nullUserLogMinutes = 60;	//log NULL user requests no more than once this often
+    static time_t lastNullUserLogEntry = (time_t)NULL;
```

Comment:
time_t is a scalar, don't think nullptr should be used here, 0 would be more correct.